### DISCUSSION TITLE
problem in place holders in my vscode

### DISCUSSION THREAD

**Discussion by SasiriChathurika** [microsoft/vscode-cpptools#7310] [General]
I have a problem in vscode
when i enter $ in my vscode it doesn't show in different colors
please help me to solve it
I'm new to Javascript
![Capture1](https://user-images.githubusercontent.com/79395595/113881886-736b2b00-97da-11eb-87b0-99d8ba43e86f.PNG)

**Comment by sean-mcmanus** [comment]
Using `""` doesn't accept newlines in strings. You can try using `` instead.

**Reply by bobbrow** [reply]
`""` also doesn't do variable substitution.  You have to use `` for that.
And just a note for others.  This is TypeScript, not C++.
